databind/src/test/java/tech/ydb/yoj/databind/schema/ChangefeedSchemaTest.java

```diff
@@ -21,6 +23,10 @@ public void testChangefeedDefaultsEntity() {
         assertThat(entitySchema.getChangefeeds()).hasSize(1);
         Assertions.assertThat(entitySchema.getChangefeeds().get(0).getMode()).isEqualTo(Changefeed.Mode.NEW_IMAGE);
         Assertions.assertThat(entitySchema.getChangefeeds().get(0).getFormat()).isEqualTo(Changefeed.Format.JSON);
+        Assertions.assertThat(entitySchema.getChangefeeds().get(0).getRetentionPeriod())
```

Comment:
minor: style: let's remove explicit class qualification (`Assertions.`) from this test, `assertThat()` method is `static import`ed already and can be used just as `assertThat(xxx).<...>`